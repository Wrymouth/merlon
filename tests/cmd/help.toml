bin.name = "merlon"
args = ["help"]
