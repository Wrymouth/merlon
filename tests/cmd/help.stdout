Mod package manager for the Paper Mario (N64) decompilation.

Merlon allows you to create mod packages by editing the decomp source code, and to export packages into distributable `.merlon` patch files that can be applied to a base ROM with `merlon apply`.

For assistance with Merlon, join the #merlon channel of the Paper Mario Modding Discord server:

https://discord.gg/paper-mario-modding-279322074412089344 https://discord.com/channels/279322074412089344/1099844075399696394

Copyright 2023 Alex Bates

This Executable Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this program, You can obtain one at https://mozilla.org/MPL/2.0/.

Usage: merlon [OPTIONS] <COMMAND>

Commands:
  new
          Create a new package
  init
          Initialise this package for editing and building
  export
          Export this package as a `.merlon` file for distribution
  apply
          Apply a distributable to a base ROM
  open
          Open a distributable's source code
  run
          Run current mod in an emulator
  build
          Build current mod into a ROM
  update
          Update all dependencies, including packages and the decomp
  help
          Print this message or the help of the given subcommand(s)

Options:
  -d, --directory <DIRECTORY>
          The directory of the package to operate on.
          
          If not set, a `merlon.toml` file will be searched for in the current directory and its parents.

  -h, --help
          Print help (see a summary with '-h')

  -V, --version
          Print version
